### PR TITLE
Refactor: path.rglob("…") does not need "**"

### PR DESCRIPTION
**Pull request by eumiro**
Replace `path.glob("**/…")` with `path.rglob("…")`. If the `**` placeholder is deeper, then extend `path`.

### CODE REVIEW

File: tests/system/providers/google/cloud/gcs/resources/transform_timespan.py [hunk 1]
```diff
@@ -28,11 +28,11 @@
...
-    for f in Path(source).glob("**/*"):
+with Path(destination, "output.txt").open("w+") as dest:
+    for f in Path(source).rglob("*"):
         if f.is_dir():
             continue
-    with open(f) as src:
-        lines = [line.upper() for line in src.readlines()]
-        print(lines)
-        dest.writelines(lines)
+        with f.open() as src:
```
Comment (eumiro) [inline tests/system/providers/google/cloud/gcs/resources/transform_timespan.py]: I think this block should be indented, so it happens for every `f` and not only for the last one. Please review.

Comment (potiuk) [reply tests/system/providers/google/cloud/gcs/resources/transform_timespan.py]: Yes. Google System tests are not **yet** run regularly. Google team works on automatically running them and providing a dashboard so they would get to this one too.. but we can correct it now.

**Review by potiuk — APPROVED** [review]

**Comment by potiuk** [comment]
conflicts/tests/ to solve

**Review by potiuk — APPROVED** [review]

**Comment by potiuk** [comment]
Conflicts now :(

**Pull request merged**